packages/remix-server-runtime/responses.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * A JSON response. Converts `data` to JSON and sets the `Content-Type` header.
  */
-export function json(data: any, init: number | ResponseInit = {}): Response {
+export function json<Data>(data: Data, init: number | ResponseInit = {}): Response {
```

Comment:
Defaulting `Data` to `unknown` makes sure we don't **have** to pass the type

```suggestion
export function json<Data = unknown>(data: Data, init: number | ResponseInit = {}): Response {
```

Reply:
That's not really needed, TS infer it because it's being used in the param (just tried again)

Reply:
Forget what I said then 🙈